.github/workflows/release.yml

```diff
@@ -93,8 +93,8 @@ jobs:
           overwrite: false
 
   typegraph:
-    needs:
-      - check-bump
+    # needs:
```

Comment:
why?

Reply:
Yeah, sorry. Disabled them to get it working with `act`. Reverted now.